ang/civicase/ActivitiesCalendar.js

```diff
@@ -397,7 +421,13 @@
         params.case_id = getCaseIdApiParam();
       }
 
-      return crmApi('Activity', 'getdayswithactivities', params);
+      return crmApi('Activity', 'getdayswithactivities', params)
+        .then(function (result) {
+          return result.values;
+        })
+        .catch(function () {
+          return [];
```

Comment:
why would this endpoint throw an error? Just asking.

Reply:
In case the user doesn't have any cases he's involved with, and he selects the "Cases I'm involved with" filter, the `case_id` param will be simply an empty array `[]`, and the endpoint won't like that. I thought about maybe dealing specifically with that case but in the end it would have been to ad-hoc to me. This approach is more generic: if something goes wrong, then simply flush all the days and don't display anything (which is what needs to happen when the user doesn't have any cases he's involved with)